src/unity/python/turicreate/toolkits/image_similarity/image_similarity.py

```diff
@@ -105,7 +105,14 @@ def create(dataset, label = None, feature = None, model = 'resnet-50', verbose =
     # Check parameters
     allowed_models = list(_pre_trained_models.MODELS.keys())
     if _mac_ver() >= (10,14):
-        allowed_models.append('VisionFeaturePrint_Screen')
+        allowed_models.append('VisionFeaturePrint_Scene')
+
+        # Also, to make sure existing code doesn't break, replace incorrect name
+        # with the correct name version
+        if model == "VisionFeaturePrint_Screen":
+            print "WARNING: Correct spelling of model name is VisionFeaturePrint_Scene."
```

Comment:
Same point here as I made in the image classifier: this will not work in Python 3 and we should mention we will remove this argument value in a later release.

Reply:
Great catch!